TAScheduler/UnitTests/tests.py

```diff
@@ -716,6 +716,58 @@ def test_max_capacity_instr(self):
             self.admin.courseTAAsmgt(temp_ta, self.tempCourse)
 
 
+class AdminGetAllSecAsgmt(TestCase):  # Kiran
```

Comment:
Should be TestAdminGetAllSectionAssignments. Need to have the word "Test" in order to run unittests